.github/workflows/build.yml

```
@@ -0,0 +1,35 @@
+name: Sonar Analysis
+on:
+  push:
+    branches:
+      - develop
+  pull_request:
+jobs:
+  build:
+    name: Analyse Module with SonarCloud
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0  # Shallow clones should be disabled for a better relevancy of analysis
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: Cache SonarCloud packages
+        uses: actions/cache@v1
+        with:
+          path: ~/.sonar/cache
+          key: ${{ runner.os }}-sonar
+          restore-keys: ${{ runner.os }}-sonar
+      - name: Cache Maven packages
+        uses: actions/cache@v1
+        with:
+          path: ~/.m2
+          key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
+          restore-keys: ${{ runner.os }}-m2
+      - name: Build and analyze
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}  # Needed to get PR information, if any
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+        run: mvn -B compile org.sonarsource.scanner.maven:sonar-maven-plugin:sonar -DskipTests
```

```suggestion
        run: mvn -B compile org.sonarsource.scanner.maven:sonar-maven-plugin:sonar
```
No need to  `-DskipTests` now that we don't use `verify` anymore

---

done